Bug: ChromaDB Fails on x64 Systems Without WSL

**Describe the bug**
Developers on **x64 systems (Intel/AMD)** will likely encounter this error when running `npm run ai:server`:

```Js
Error: Cannot find module 'chromadb-js-bindings-linux-x64-gnu' 
````
This happens because **ChromaDB bindings are architecture-dependent** and the setup process currently assumes an ARM64 environment.
As a result, contributors using x64 Ubuntu or Windows cannot start the AI server directly.

**Impact**
- Blocks contributors using standard x64 hardware
- Prevents local testing of AI-related features
- Creates setup friction during onboarding and Hacktoberfest participation

**To Reproduce**
Steps to reproduce the behavior:
1. Clone the repository on an x64 system (Ubuntu 22.04, WSL2, or Intel-based environment)
2. Run `npm install`
3. Attempt to start the AI server: `npm run ai:server`
4. Observe the module error

**Expected behavior**
The AI server should start successfully on x64 systems with ChromaDB bindings properly installed for the detected architecture.

**Actual Behavior**
The server crashes immediately with a missing module error, suggesting the correct x64 bindings were not installed or linked during `npm install`.


**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04 LTS / Windows 10/11 (x64)
 - Node.js: v20.19.5
 - npm: 10.8.2
 - Architecture: x86_64 (x64)

**Confirmed Workaround**
**Using WSL (Windows Subsystem for Linux)** with Ubuntu 22.04 LTS resolves the issue completely.

Developers can install Neo within WSL, open it in **VS Code** via the **WSL extension (Microsoft)**, and run:

```bash
   npm install
   npm run build-all
   npm run ai:server
````
✅ The AI server launches successfully under this setup.

**Suggested Documentation Update**

Add a new markdown file:
📄 `.github/DOCS/setup-x64-wsl-guide.md`

This file should include:
- Instructions for enabling virtualization in BIOS
- WSL installation and Ubuntu setup
- VS Code configuration (with WSL extension)
- Steps to copy existing forks into WSL safely
- Commands to verify ChromaDB runs correctly

This keeps the main **README** lightweight while providing detailed help for x64 users.

Labels

`setup`, `bug`, `documentation`, `hacktoberfest`, `good first issue`

@tobiu , I'd love to work on this issue.

--- a/.github/AI_TOOLING_WSL_SETUP.md
+++ b/.github/AI_TOOLING_WSL_SETUP.md
@@ -1,6 +1,9 @@
 # Getting Started with Neo on Windows
 
-Welcome! This guide will help you set up Neo on Windows using WSL (Windows Subsystem for Linux). Don't worry if you're new to this, we'll walk through everything step by step.
+Welcome! This guide provides a workaround for a known installation issue with **ChromaDB** on **x64 Windows** machines when setting up your AI development environment. 
+
+It’s designed to help you get things running smoothly, but note that this isn’t a general requirement for using the Neo.mjs framework. Don’t worry if you’re new, we’ll go through each step clearly and simply.
+
 
 ## What You'll Need
 

